.editorconfig

```diff
@@ -14,6 +14,10 @@ indent_size = 4
 indent_style = space
 indent_size = 4
 
+[test/**/schema.json]
```

Comment:
I added this rule because the editor kept changing the indent of `schema.json` files to 4.